java-symbolic-execution/src/test/java/org/sonar/java/se/xproc/BehaviorCacheTest.java

```diff
@@ -107,15 +111,15 @@ void compute_behavior_only_once() throws Exception {
 
   @Test
   void explore_method_with_recursive_call() throws Exception {
-    SymbolicExecutionVisitor sev = createSymbolicExecutionVisitor("src/test/resources/se/RecursiveCall.java",
+    SymbolicExecutionVisitor sev = createSymbolicExecutionVisitor("src/test/resources/se/RecursiveCall.java", 
```

Comment:
Still two little formatting changes that we don't need in this file (Line 114 and 122).